index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <link rel="stylesheet" href="./assets/css/style.css" />
+        <title>Elizabeth's Portfolio Page</title>
+    </head>
+  <body>
+      <!--navigation bar of the web page-->
+      <header class="header">
```

Comment:
Your VS Code doesn't seem to have formatted your HTML, so it's a little tricky to see which opening and closing tags belong to each other. I think this is because you have a couple of errors in your code (see comments below about tags with don't have closing elements). Once you fix these, the formatting should fix itself, but make sure you are checking your Prettier for errors - please reach out if you are unsure what I mean by this!